internal/pkg/template/templates/cicd/partials/action-config.yml

```diff
@@ -0,0 +1,23 @@
+{{- if or (eq .Build.EnvironmentType "LINUX_CONTAINER") (eq .Build.EnvironmentType "ARM_CONTAINER")}}
```

Comment:
Can we also reuse for the current pipeline CFN build?

Reply:
👍🏼  Just to confirm, do you mean having the current pipeline CFN build project (i.e. build-action.yml) to include this snippet?

```yaml
BuildProject:
  Properties:
     {{- include "action-config" . | indent 6}}
```

I will make ⬆️  change, but i want to make sure that we are on the same page!